### ISSUE TITLE
CUSA00433 - Diablo III: Reaper of Souls - Ultimate Evil Edition

### ISSUE THREAD

**Issue by kanelakis** [shadps4-emu/shadps4-game-compatibility#1563]
### Game Name

Diablo III: Reaper of Souls - Ultimate Evil Edition

### Game code

CUSA00433

### Game version

v01.46

### Used emulator's version (only released versions are acceptable)

v0.5.0

### Current status

Boots

### Operating System

Windows

### Error

[Error.txt](https://github.com/user-attachments/files/18261125/Error.txt)

### Description

The game starts up to that point of the photos, then pressing the circle button crashes and closes the emulator.

### Screenshots

![Screenshot 2024-12-27 153908](https://github.com/user-attachments/assets/ba9823d3-ad1c-48a9-b211-634fe275bec8)
![Screenshot 2024-12-27 153919](https://github.com/user-attachments/assets/42fe2624-b74c-484d-9fc1-fb41d5106f4e)
![Screenshot 2024-12-27 153932](https://github.com/user-attachments/assets/8dfc5010-ef70-4527-a5fa-0c495644dc23)

### Log File

[Diablo III log.txt](https://github.com/user-attachments/files/18261147/Diablo.III.log.txt)

**Comment by Hermiten** [comment]
Can you move on the menu or is it before the menu ?

**Comment by kanelakis** [comment]
> Can you move on the menu or is it before the menu ?

No, it crashes with the last photo, then it gives the purple error message at the end and the emulator closes.
It doesn't get to the menu to select characters.

**Comment by kanelakis** [comment]
[shad_log.txt](https://github.com/user-attachments/files/18626927/shad_log.txt)
Emulator v0.6.0
The game crashes and closes on autosave, or sometimes when you are about to select the character.

**Comment by gandalfthewhite19890404** [comment]
Try to use 1.00. I'm using CUSA00434, same as 433, just different language - playable with some graphics bugs, noticeable in town, already got to Act 2 with other character.

![Image](https://github.com/user-attachments/assets/1f966545-7019-459c-9585-2756ad68f4d1)

![Image](https://github.com/user-attachments/assets/8c0bc980-ea04-4405-b5b6-8f5ff89e3582)

**Comment by brad0demx** [comment]
Since the CUSA is different in gandalfthewhite's reply, @kanelakis could you please try 1.00 and report back so I can update the status on this serial.

**Comment by kanelakis** [comment]
> Since the CUSA is different in gandalfthewhite's reply, [@kanelakis](https://github.com/kanelakis) could you please try 1.00 and report back so I can update the status on this serial.

[shad_log.txt](https://github.com/user-attachments/files/18629238/shad_log.txt)
Hi then I removed the update and the dlc thinking they were the cause.
I also put the compatibility modules 11, but nothing has changed it freezes at autosave or while entering the character menu.

**Comment by brad0demx** [comment]
Thank you for testing it again I will leave this report as menus for now. @gandalfthewhite19890404 You could open an issue for your CUSA

**Comment by kanelakis** [comment]
I tested with version 0.6.1 131b6f9.
It stayed in the character menu for a while and then it crashed.

[shad_log.txt](https://github.com/user-attachments/files/18656748/shad_log.txt)
